scripts/corelli/calibration/utils.py

```diff
@@ -75,6 +78,57 @@ def bank_numbers(bank_selection: str) -> List[str]:
     return [str(n) for n in sorted(banks)]
 
 
+def preprocess_banks(input_workspace: str, output_workspace: str) -> Workspace2D:
+    r"""
+    Clone the input workspace/run and preprocess the histograms for each tube such
+    that the peak finding algorithm can have a better chance of finding the correct
+    wire position.
+
```

Comment:
Can you include a two-liner summarizing what the preprocessing consist of?

Reply:
added additional information on how preprcess is performed